### ISSUE TITLE
Deprecate `date to-record` and `date to-table`

### ISSUE THREAD

**Issue by KAAtheWiseGit** [nushell/nushell#11234]
I suggest removing `date to-record` command in favor of `into record`, by moving the output of `date to-record` to `<date> | into record`, as it returns nanoseconds.

And removing `date to-table` in favor of `2020-04-12T22:10:57+02:00 | into record | [$in]`.

**Comment by fdncred** [comment]
I'd support this as long as there were examples that demonstrate the usage, and like you say, the date to-record format is moved over to into record.

**Comment by KAAtheWiseGit** [comment]
`into record` has a `2020-04-12T22:10:57+02:00 | into record` example.

Unsure where the example for `date to-table` can be added.  There is currently no `into table`, which could in theory be changed.

**Comment by amtoine** [comment]
yeah, if we add the nanoseconds to the date, i support this deprecation as they can both be achieved with `into record` and `into record | [$in]` or `into record | transpose | transpose -r` :+1:

**Comment by LoicRiegel** [comment]
This seems to be done, can this be closed?
